lib/setup/setupRepo.js

```diff
@@ -99,7 +99,7 @@ function Repo(options) {
                 text += '"' + name + '"' + ((name.length < 15) ? new Array(15 - name.length).join(' ') : '');
 
                 var tLen = 10;
-                if (name.length >= 15) tLen -= (name.length > tLen);
+                if (name.length >= 15) tLen -= (name.length > tLen ? 1 : 0);

```

Comment:
This syntax was unclear as we subtract a boolean from a number.

Reply:
makes sense